release_creation/snapshot/download_no_deps.sh

```diff
@@ -10,4 +11,14 @@ pip download -r $requirements_file \
  --platform $platform \
  --no-deps
 
+# some mac builds needs wheel and cython installed
+# include the psycopg2-binary in mac builds for ease of install
+if [[ "$OSTYPE" == darwin* ]]; then 
+    pip download wheel cython psycopg2-binary==2.9.5 \
+    --dest $staging \
```

Comment:
Make sure to quote these variables when referencing. Really, we should get a linter setup because it'll complain about these things.